第四次实验/task2.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
```

Comment:
In task2.sh line 11:
            ColNum=$(($ColNum+1))
                      ^-- SC2004: $/${} is unnecessary on arithmetic variables.


In task2.sh line 36:
            below20=$(($below20+1))
                       ^-- SC2004: $/${} is unnecessary on arithmetic variables.


In task2.sh line 42:
            above30=$(($above30+1))
                       ^-- SC2004: $/${} is unnecessary on arithmetic variables.


In task2.sh line 48:
            between2030=$(($between2030+1))
                           ^-- SC2004: $/${} is unnecessary on arithmetic variables.


In task2.sh line 50:
        count=$(($count+1))
                 ^-- SC2004: $/${} is unnecessary on arithmetic variables.


In task2.sh line 53:
    count=$(($count-1))
             ^-- SC2004: $/${} is unnecessary on arithmetic variables.


In task2.sh line 76:
        count=$(($count+1))
                 ^-- SC2004: $/${} is unnecessary on arithmetic variables.


In task2.sh line 79:
    for key in ${!dic[@]}; do 
               ^-- SC2068: Double quote array expansions, otherwise they're like $* and break on spaces.


In task2.sh line 88:
    nameLength=$( awk -F '\t' "{print length($tmp)}" $1)
                                                     ^-- SC2086: Double quote to prevent globbing and word splitting.


In task2.sh line 102:
        count=$(($count+1))
                 ^-- SC2004: $/${} is unnecessary on arithmetic variables.


In task2.sh line 111:
    echo "Usage: bash "$0" [options]"
                       ^-- SC2027: The surrounding quotes actually unquote this. Remove or escape them.
                       ^-- SC2086: Double quote to prevent globbing and word splitting.